Source/r_sky.c

```diff
@@ -67,6 +69,114 @@ void R_InitSkyMap (void)
   skytexturemid = 100*FRACUNIT;
 }
 
+static byte V_GetPaletteIndex(byte *palette, int r, int g, int b)
```

Comment:
This function taken from Chocolate Doom's `i_video.c` (I think). Could you please document this as before for `R_DrawSkySpan()`?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/483/commits/7dc98d16ed4873ace4264305216c4135343ec9c4